setup-remote.md

```diff
@@ -0,0 +1,233 @@
+`NETWORK SERVICE` account permissions to read the private key from the
+certificate. Follow the instructions in the section above on `Granting
+permissions to read the SSL certificate's private key` to complete this step.
+
+## Installing R Services
+
+Now that you have your SSL certificate installed, and you have granted the NETWORK
+SERVICE account access to its private key, you are now ready to install the R Services.
+
+First, you will need to install an R interpreter on your R server computer. You can
+choose from Microsoft R Open or CRAN R. Both have identical functionality, but
+Microsoft R Open benefits from additional hardware accelerated linear algebra 
+libraries courtesy of the [Intel Math Kernel Library](https://software.intel.com/en-us/intel-mkl).
+
+You can download R from one of the links below:
+
+* [Microsoft R Open](https://mran.microsoft.com/open/)
+* [CRAN R for Windows](https://cran.r-project.org/bin/windows/base/)
+
+Once you have completed installing R on the R server computer, you can install the
+R components. 
+
+* Run the [R Services installer](https://aka.ms/rtvs-services).
+
+When the installer is complete, it will ask you to reboot the machine. Once the
+machine has completed rebooting, the R services will automatically start. You can 
+see the services using the Services management console snap-in. Launch it from
+an administrator command prompt using `compmgmt.msc`:
+
+![Services management console](./media/services_mmc.png)
+
+As you can see, there are two services running. The `R Host Broker Service` is
+the component that handles all HTTPS traffic between Visual Studio and process
+where the R code runs on the machine. The `R User Profile Service` is a
+privileged component that handles Windows user profile creation. This is called
+when a new user first logs onto the R server machine.
+
+## Configuring R Services
+
+### User accounts
+
+Now that you have installed R services on the machine, you'll need to create
+some user accounts for your users. You can either create standard
+(non-privileged) local user accounts for your users, or you can join your R
+server machine to your domain and add the appropriate security groups to the
+`Users` security group on your R server.
+
+### Firewall rules
+
+By default, the `R Host Broker` listens on TCP port 5444. Therefore, you must
+ensure that there are Windows firewall rules enabled that will let traffic in
+(and enable outbound traffic as well for scenarios like installing packages).
+The installer for R services will do this automatically for you. However, if you
+are using 3rd party firewall software, you will need to configure it to allow R
+Host Broker traffic to pass through. 
+
+### Azure configuration
+
+Azure networking also implements its own firewall that is independently
+configured from the Windows firewall. You will need to make sure that you have
+the appropriate set of rules in place to allow incoming traffic through. You
```

Comment:
We probably need to add section on the cert name since in Azure can you had to add specific cert name to the broker .json config file